css/styles.css

```diff
@@ -117,7 +116,7 @@ input:focus {
 .card {
   display: inline-block;
   padding: 30px;
-  margin: 5rem;
+  margin: 2rem;
```

Comment:
The smaller margins look good in terms of width. I would, however, keep the larger margins on the top and bottom so that the cards are portrait oriented.

Reply:
Sorry. Extra _padding_ needs to be added in order to make the card orientation portrait.
Maybe try something along the lines of `padding: 100px 60px;`?